### DISCUSSION TITLE
Duplicate IP

### DISCUSSION THREAD

**Discussion by MrgampRus** [bb-Ricardo/netbox-sync#307] [Q&A]
Good afternoon, I noticed that if the IP address exists in Netbox, after running the script, the IP address is duplicated. Can you tell me if this is normal behavior?

**Comment by bb-Ricardo** [comment]
This depends on the scope the IP address or a coresponding subnet exists. If you have created this IP on the "Global" VRF but the site RFV contains a matching prefix, then this IP address will be created twice in different VRF. just run the script in DEBUG2 log mode and it should tell you on which bases this IP address has been added.
